lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -159,9 +205,9 @@ class LighthouseReportViewer {
       renderer.renderReport(json, container);
 
       // Only give gist-saving callback (and clear gist from query string) if
-      // current report isn't from a gist.
+      // current report isn't from a gist or PSI.
```

Comment:
seems like it would be good to allow this?

Reply:
true. what i didn't want was the history modification.

Reply:
fixed